<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>DialogEditLabel</class>
 <widget class="QDialog" name="DialogEditLabel">
  <property name="windowModality">
   <enum>Qt::ApplicationModal</enum>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>437</width>
    <height>464</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Edit label template</string>
  </property>
  <property name="windowIcon">
   <iconset resource="../../../vmisc/share/resources/icon.qrc">
    <normaloff>:/icon/64x64/icon64x64.png</normaloff>:/icon/64x64/icon64x64.png</iconset>
  </property>
  <property name="modal">
   <bool>true</bool>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_2">
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_9">
     <item>
      <widget class="QToolButton" name="toolButtonNewLabel">
       <property name="enabled">
        <bool>false</bool>
       </property>
       <property name="toolTip">
        <string>Clear current and begin new label</string>
       </property>
       <property name="text">
        <string notr="true">...</string>
       </property>
       <property name="icon">
        <iconset theme="document-new">
         <normaloff>.</normaloff>.</iconset>
       </property>
       <property name="iconSize">
        <size>
         <width>24</width>
         <height>24</height>
        </size>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer_3">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QToolButton" name="toolButtonImportLabel">
       <property name="toolTip">
        <string>Import from label template</string>
       </property>
       <property name="text">
        <string notr="true">...</string>
       </property>
       <property name="icon">
        <iconset theme="document-open">
         <normaloff>.</normaloff>.</iconset>
       </property>
       <property name="iconSize">
        <size>
         <width>24</width>
         <height>24</height>
        </size>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QToolButton" name="toolButtonExportLabel">
       <property name="enabled">
        <bool>false</bool>
       </property>
       <property name="toolTip">
        <string>Export label as template</string>
       </property>
       <property name="text">
        <string notr="true">...</string>
       </property>
       <property name="icon">
        <iconset theme="document-save-as">
         <normaloff>.</normaloff>.</iconset>
       </property>
       <property name="iconSize">
        <size>
         <width>24</width>
         <height>24</height>
        </size>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QTabWidget" name="tabWidget">
     <property name="currentIndex">
      <number>0</number>
     </property>
     <widget class="QWidget" name="tabEdit">
      <attribute name="title">
       <string>Edit</string>
      </attribute>
      <layout class="QVBoxLayout" name="verticalLayout">
       <item>
        <widget class="QListWidget" name="listWidgetEdit">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
           <horstretch>0</horstretch>
           <verstretch>1</verstretch>
          </sizepolicy>
         </property>
         <property name="dragDropMode">
          <enum>QAbstractItemView::InternalMove</enum>
         </property>
        </widget>
       </item>
       <item>
        <layout class="QFormLayout" name="formLayout">
         <item row="0" column="1">
          <layout class="QHBoxLayout" name="horizontalLayout_11">
           <property name="spacing">
            <number>6</number>
           </property>
           <item>
            <widget class="QToolButton" name="toolButtonBold">
             <property name="enabled">
              <bool>false</bool>
             </property>
             <property name="toolTip">
              <string comment="Font formating">Bold</string>
             </property>
             <property name="text">
              <string notr="true">...</string>
             </property>
             <property name="icon">
              <iconset theme="format-text-bold">
               <normaloff>.</normaloff>.</iconset>
             </property>
             <property name="checkable">
              <bool>true</bool>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QToolButton" name="toolButtonItalic">
             <property name="enabled">
              <bool>false</bool>
             </property>
             <property name="toolTip">
              <string comment="Font formating">Italic</string>
             </property>
             <property name="text">
              <string notr="true">...</string>
             </property>
             <property name="icon">
              <iconset theme="format-text-italic">
               <normaloff>.</normaloff>.</iconset>
             </property>
             <property name="checkable">
              <bool>true</bool>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QToolButton" name="toolButtonTextLeft">
             <property name="enabled">
              <bool>false</bool>
             </property>
             <property name="toolTip">
              <string>Aligns with the left edge</string>
             </property>
             <property name="text">
              <string notr="true">...</string>
             </property>
             <property name="icon">
              <iconset theme="format-justify-left">
               <normaloff>.</normaloff>.</iconset>
             </property>
             <property name="checkable">
              <bool>true</bool>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QToolButton" name="toolButtonTextCenter">
             <property name="enabled">
              <bool>false</bool>
             </property>
             <property name="toolTip">
              <string>Centers horizontally in the available space</string>
             </property>
             <property name="text">
              <string notr="true">...</string>
             </property>
             <property name="icon">
              <iconset theme="format-justify-center">
               <normaloff>.</normaloff>.</iconset>
             </property>
             <property name="checkable">
              <bool>true</bool>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QToolButton" name="toolButtonTextRight">
             <property name="enabled">
              <bool>false</bool>
             </property>
             <property name="toolTip">
              <string>Aligns with the right edge</string>
             </property>
             <property name="text">
              <string notr="true">...</string>
             </property>
             <property name="icon">
              <iconset theme="format-justify-right">
               <normaloff>.</normaloff>.</iconset>
             </property>
             <property name="checkable">
              <bool>true</bool>
             </property>
            </widget>
           </item>
           <item>
            <spacer name="horizontalSpacer_5">
             <property name="orientation">
              <enum>Qt::Horizontal</enum>
             </property>
             <property name="sizeHint" stdset="0">
              <size>
               <width>20</width>
               <height>20</height>
              </size>
             </property>
            </spacer>
           </item>
           <item alignment="Qt::AlignRight">
            <widget class="QToolButton" name="toolButtonAdd">
             <property name="sizePolicy">
              <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
               <horstretch>0</horstretch>
               <verstretch>0</verstretch>
              </sizepolicy>
             </property>
             <property name="text">
              <string notr="true">...</string>
             </property>
             <property name="icon">
              <iconset theme="list-add">
               <normaloff>../../../../app/valentina/dialogs</normaloff>../../../../app/valentina/dialogs</iconset>
             </property>
            </widget>
           </item>
           <item alignment="Qt::AlignLeft">
            <widget class="QToolButton" name="toolButtonRemove">
             <property name="enabled">
              <bool>false</bool>
             </property>
             <property name="text">
              <string notr="true">...</string>
             </property>
             <property name="icon">
              <iconset theme="list-remove">
               <normaloff>../../../../app/valentina/dialogs</normaloff>../../../../app/valentina/dialogs</iconset>
             </property>
            </widget>
           </item>
          </layout>
         </item>
         <item row="1" column="0">
          <widget class="QLabel" name="label_11">
           <property name="text">
            <string>Text:</string>
           </property>
          </widget>
         </item>
         <item row="1" column="1">
          <layout class="QHBoxLayout" name="horizontalLayout_6">
           <item>
            <widget class="VLineEdit" name="lineEditLine">
             <property name="enabled">
              <bool>false</bool>
             </property>
             <property name="placeholderText">
              <string>Line of text</string>
             </property>
             <property name="clearButtonEnabled">
              <bool>false</bool>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QPushButton" name="pushButtonInsert">
             <property name="enabled">
              <bool>false</bool>
             </property>
             <property name="toolTip">
              <string>Insert placeholders</string>
             </property>
             <property name="text">
              <string>Insert...</string>
             </property>
            </widget>
           </item>
          </layout>
         </item>
        </layout>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="tabPreview">
      <attribute name="title">
       <string>Preview</string>
      </attribute>
      <layout class="QVBoxLayout" name="verticalLayout_3">
       <item>
        <widget class="QListWidget" name="listWidgetPreview"/>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
   <item>
    <widget class="QDialogButtonBox" name="buttonBox">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="standardButtons">
      <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>VLineEdit</class>
   <extends>QLineEdit</extends>
   <header>vlineedit.h</header>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="../../../vmisc/share/resources/icon.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>DialogEditLabel</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>248</x>
     <y>254</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>DialogEditLabel</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>316</x>
     <y>260</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
